public/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
@malvena15 Do we need this file?

Reply:
The manifest.json is not obligatory, but if we use API WebExtension we need that file